src/Microsoft.Security.Utilities.Core/IdentifiableScan.cs

```diff
@@ -123,7 +128,18 @@ public void Start()
     {
         if (this.scan == IntPtr.Zero)
         {
-            this.scan = identifiable_scan_create();
+            stringBuilder ??= new StringBuilder();
```

Comment:
>StringBuilder [](http://example.com/codeflow?start=34&length=13)

this is exactly what i asked for so why would i complain? but on second reflection, probably we could have just had a per-instance item here. 

but let's leave it, it's not worth going back. thanks for acting on the notes.